### DISCUSSION TITLE
還在煩惱怎麼處理系統相關問題，讓OS 幫助你

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#940] [優秀Python套件]
我個人最喜歡os系統內的功能就是listdir 和 endwith 如果使用這個語法就可以自動遍歷資料夾內的特定檔案
範例:    file = [_ for _ in os.listdir(ahead) if _.endswith(fileExt)]
想知道其他用法的可以點擊https://docs.python.org/zh-tw/3/library/os.html

**Comment by Evan0979** [comment]
今天早上剛考完,作業系統真的難QQ

**Comment by CPCpyt** [comment]
又學到了新知識！感謝分享！
雖然目前還沒有運用到但我會留著日後學習！
